.rubocop.yml

```diff
@@ -64,3 +64,7 @@ Govuk/GovukLinkTo:
     # link_to for kaminari pagination links
     - 'app/views/kaminari/_next_page.html.erb'
     - 'app/views/kaminari/_prev_page.html.erb'
+
+RSpec/FilePath:
+  Exclude:
+    - 'spec/presenters/vendor_api/**/*'
```

Comment:
Can we move this here: https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/config/rubocop/config/rspec.yml